extractors/python_functions/color_code_extraction/code_snippet.md

```diff
@@ -1,11 +1,10 @@
 ```python
 import re
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "color"
 
-def my_tagger(record) -> List[Tuple[str, int, int]]:
+def my_tagger(record):
```

Comment:
my in function name seems a bit odd. usually the name of the function represents the tasks to some extend so I would go with `color_extractor` I think

Reply:
Changed!